infrastructure/helm/keycloak/quantumserverless-kuberay-patch.yaml

```diff
@@ -0,0 +1,35 @@
+spec:
+  headGroupSpec:
```

Comment:
Why are we adding arguments to `headNode` the same that we are doing in the `sidecarContainers`?

Reply:
It's the way I created the patch file.  We can eliminate the patch process.

Reply:
At least following a similar approach, thinking that the user can configure the release by itself too (so it can obtain that info from its infra by itself).